superset/tasks/schedules.py

```diff
@@ -569,7 +583,7 @@ class AlertState:
 
 
 def deliver_alert(alert_id: int, recipients: Optional[str] = None) -> None:
-    alert = db.session.query(Alert).get(alert_id)
+    alert = db.create_scoped_session().query(Alert).get(alert_id)
```

Comment:
This is interesting, @john-bodley submitted this: https://github.com/apache/incubator-superset/pull/10427

is this necessary because it's called by a celery task, and we want to make sure the session gets removed on task completion? What's the effect if this is called by a web request with proper session scope?

Reply:
Without scoped_session errors like this https://github.com/apache/incubator-superset/issues/10530 are more prevelant. A scoped session is able to bring down the amount of errors but not completely get rid of them. I believe https://github.com/apache/incubator-superset/pull/10537 was submitted because the change caused more problems than it fixed.

Reply:
For now I'm actually going to revert this change and just use regular session. Both types of sessions cause errors with the underlying cause remaining unknown for now.